### PR TITLE
Include handlers in Admin API Version create call

### LINKED ISSUE #682: Include handlers in version create calls
As it is, the tools omit the list of configured handlers when making Admin API calls, requiring the controller to parse the version source. It would be better to keep them together with the rest of the version configuration options.

### PR DESCRIPTION
**Pull request by cdonati**
Resolves #682.

### CODE REVIEW

File: appscale/tools/admin_api/handler.py [hunk 1]
```diff
@@ -0,0 +1,157 @@
+""" Represents an Admin API UrlMap resource. """
+
+import re
+
+from appscale.tools.custom_exceptions import AppEngineConfigException
+
+DELTA_REGEX = r'([0-9]+)([DdHhMm]|[sS]?)'
+
+EXPIRATION_RE = r'\s*({})(\s+{})*\s*'.format(DELTA_REGEX, DELTA_REGEX)
```
Comment (sjones4) [inline appscale/tools/admin_api/handler.py]: Should this be for a full match?, i.e. ^...$

File: appscale/tools/admin_api/handler.py [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+
+  API_VALUES = {
+    'securityLevel': {'optional': 'SECURE_OPTIONAL',
+                      'never': 'SECURE_NEVER',
+                      'always': 'SECURE_ALWAYS'},
+    'login': {'optional': 'LOGIN_OPTIONAL',
+              'required': 'LOGIN_REQUIRED',
+              'admin': 'LOGIN_ADMIN'},
+    'authFailAction': {'redirect': 'AUTH_FAIL_ACTION_REDIRECT',
+                       'unauthorized': 'AUTH_FAIL_ACTION_UNAUTHORIZED'}
```
Comment (sjones4) [inline appscale/tools/admin_api/handler.py]: should redirectHttpResponseCode be here also, i.e.

```
    'redirectHttpResponseCode': {301: 'REDIRECT_HTTP_RESPONSE_CODE_301',
                                 302: 'REDIRECT_HTTP_RESPONSE_CODE_302',
                                 303: 'REDIRECT_HTTP_RESPONSE_CODE_303',
                                 307: 'REDIRECT_HTTP_RESPONSE_CODE_307'}
```

File: appscale/tools/admin_api/handler.py (hunk 1 above)
Comment (cdonati) [reply appscale/tools/admin_api/handler.py]: Yes. I copied the regex from the SDK without much thought. That addition makes it a lot stricter.

File: appscale/tools/admin_api/handler.py (hunk 2 above)
Comment (cdonati) [reply appscale/tools/admin_api/handler.py]: Yes. Good catch.

**Comment by cdonati** [comment]
https://ocd.appscale.com:8080/job/Daily%20Build/5439/

**Pull request merged**